packages/std/packages/manifest_tool/src/lib.rs

```diff
@@ -0,0 +1,202 @@
+pub use file::File;
+use num::ToPrimitive;
+use std::{io::Read, ops::Range, path::Path};
+use zerocopy::{FromZeros as _, IntoBytes as _};
+
+mod elf;
+mod file;
+mod mach;
+
+#[derive(
+	Debug,
+	Clone,
+	Copy,
+	zerocopy::FromBytes,
+	zerocopy::IntoBytes,
+	zerocopy::Immutable,
+	zerocopy::KnownLayout,
+)]
+#[repr(C)]
+struct Footer {
+	pub size: u64,
+	pub version: u64,
+	pub magic: [u8; 8],
+}
+
+const MAGIC: [u8; 8] = *b"tangram\0";
```

Comment:
tiny tiny nit - this is exported in the std lib in `manifest.rs`, which is in the workspace. You aren't currently pulling this in, so maybe better just to duplicate.